ops/docker/Dockerfile.packages

```diff
@@ -1,10 +1,56 @@
 # This Dockerfile builds all the dependencies needed by the monorepo, and should
 # be used to build any of the follow-on services
 #
-# ### BASE: Install deps
+
+# ░██████╗████████╗░█████╗░░██████╗░███████╗  ░█████╗░
```

Comment:
I have a preference for removing the ascii art because it makes it more difficult to see what is in the file on a single screen

Reply:
The giant space between stages is basically a workaround to not having modules in being able to put dockerfiles together